components/console/src/schema/operation-routes.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod'
+import { metadata } from './metadata'
+
+const title = z.string().min(1).max(50)
+
+const description = z.string().max(250).optional()
+
+const operationType = z.enum(['source', 'destination'])
+
+const ruleType = z.enum(['alias', 'account_type'])
+
+const validIf = z.union([
+  z.string(),
+  z.array(z.string()).min(1, "At least one account type must be selected")
+]).optional()
+
+const account = z.object({
+  ruleType,
+  validIf
+}).nullable()
+
+const id = z.string().uuid()
+
+export const operationRoutes = {
+  id,
+  title,
+  description,
+  operationType,
+  account,
+  metadata
+}
+
+export const createOperationRouteSchema = z.object({
```

Comment:
Don't create the Zod object inside the schema file.
Export those properties as a plain object, and them create this object at the place of usage.
Follow the other schema files convention.